shuffle/cli/src/node.rs

```diff
@@ -44,6 +44,7 @@ fn create_node(home: &Home, genesis: Option<String>) -> Result<()> {
     diem_node::load_test_environment(
         Some(PathBuf::from(home.get_node_config_path())),
         false,
+        true,
```

Comment:
This LOC is only invoked on node creation. When starting and existing node (below on L81), it doesn't look like we're in `lazy` mode anymore. Shall we add the following to `start_node`?

```rust
    if lazy {
        println!("\tLazy mode is enabled");
        config.consensus.mempool_poll_count = u64::MAX;
        println!();
    }
```

Reply:
Discussed offline. We will improve the saving of test node creation in a subsequent PR.